javascripts/content_scripts/embeded.app.js

```diff
@@ -0,0 +1,367 @@
+/**
+ * @fileOverview This file handles the communication between
+ * the Privly embeded-posting app and content scripts.
+ *
```

Comment:
ok.. I mixed seamless posting and embeded posting together.. the name is confusing
I will separate them